### ISSUE TITLE
turning monitor on crashes all open WSLg windows

### ISSUE THREAD

**Issue by brenol** [microsoft/wslg#542]
# Environment

```none
Windows build number: Microsoft Windows [versão 10.0.22483.1011]
Your Distribution version: Release:        18.04 
Your WSLg version: 1.0.27
```

# Steps to reproduce
- Open Tilix/Kitty
- Turn on a monitor
- Tilix/Kitty crashes

The attached logs are printed **when** the monitor turns on.

**WSL logs**: 
* Attach WSLg logs from  `/mnt/wslg`

* no new logs in pulseaudio.log.
(had tail -f plugged on weston.log to get only new logs)
[weston.log](https://github.com/microsoft/wslg/files/7430702/weston.log)
[versions.txt](https://github.com/microsoft/wslg/files/7430697/versions.txt)

#  Expected behavior

Expected WSLg opened software to not crash

# Actual behavior

Software opened by WSLg is crashing

(sorry if its a duplicate - searched for it but I did not find a similar issue)

**Comment by hideyukn88** [comment]
@brenol, thanks for reporting the issue. Do you see any dumps are created at /mnt/wslg/dumps? if so, please share them with us, thanks!

**Comment by brenol** [comment]
Yes @hideyukn88, sorry, forgot to respond here. Do you have any suggestion of where I can attach that dump file? It's 317MB.

Thanks

**Comment by hideyukn88** [comment]
@brenol, is it after ZIP compressing? if so, please share the file with us using Feedback Hub app in Windows, it allows to attach files to the issue. Please let us know the link to the issue once report is filed, thanks!

**Comment by brenol** [comment]
Thank you!

Here it goes:
https://aka.ms/AAeq5t0
